model-zoo/src/main/java/ai/djl/basicmodelzoo/cv/classification/MobileNetV2.java

```diff
@@ -0,0 +1,410 @@
+ * https://github.com/weiaicunzai/pytorch-cifar100/blob/master/models/mobilenetv2.py (Original
+ * author weiaicunzai).
+ *
+ * <p>see https://arxiv.org/pdf/1801.04381.pdf for more information about MobileNetV2
+ */
+public final class MobileNetV2 {
+
+    public static final int FILTERLENGTH = 9;
+    public static final int REPEATLENGTH = 9;
+    public static final int STRIDELENGTH = 9;
+    public static final int MULTILENGTH = 7;
+
+    private MobileNetV2() {}
+    /**
+     * Builds a {@link Block} that represent an inverted residual Unit used in the implementation of
+     * the MobileNetV2 Model.
+     *
+     * @param inputChannels number of inputChannels of the block
+     * @param outputChannels number of outputChannels of the block
+     * @param stride control the stride of a depthWise kernel
+     * @param t the multiTime of the first pointWise Block
+     * @param batchNormMomentum the momentum of batchNormLayer
+     * @return a {@link Block} that represent an inverted residual Unit
+     */
+    public static Block linearBottleNeck(
+            int inputChannels, int outputChannels, int stride, int t, float batchNormMomentum) {
+        SequentialBlock block = new SequentialBlock();
+        block.add(
+                        Conv2d.builder() // PointWise
+                                .setFilters(inputChannels * t)
+                                .setKernelShape(new Shape(1, 1))
+                                .optBias(false)
+                                .build())
+                // add a batchNorm
+                .add(BatchNorm.builder().optEpsilon(2E-5f).optMomentum(batchNormMomentum).build())
+                // add a relu
+                .add(Activation.reluBlock())
+                .add(
+                        Conv2d.builder() // DepthWise
+                                .setKernelShape(new Shape(3, 3))
+                                .setFilters(inputChannels * t)
+                                .optStride(new Shape(stride, stride))
+                                .optPadding(new Shape(1, 1))
+                                .optGroups(inputChannels * t)
+                                .optBias(false)
+                                .build())
+                .add(BatchNorm.builder().optEpsilon(2E-5f).optMomentum(batchNormMomentum).build())
+                .add(Activation.reluBlock())
+                .add(
+                        Conv2d.builder() // PointWise
+                                .setFilters(outputChannels)
+                                .setKernelShape(new Shape(1, 1))
+                                .optBias(false)
+                                .build())
+                .add(BatchNorm.builder().optEpsilon(2E-5f).optMomentum(batchNormMomentum).build());
+
+        // if dimMatch,then add x
+        if (stride == 1 && inputChannels == outputChannels) {
+            block =
+                    new SequentialBlock()
```

Comment:
You don't need to wrap this in a SequentialBlock here. You can just directly return the ParallelBlock

Reply:
Yeah... At first I thought that we need to add a relu block at last, but I go through the paper again and find that a relublock is not needed...